newsplease/crawler/simple_crawler.py

```diff
@@ -1,7 +1,74 @@
+# Based on https://github.com/adbar/trafilatura/blob/master/trafilatura/utils.py
+import logging
+import socket
 import copy
 import threading
 
-from six.moves import urllib
+try:
+    # this module is faster
```

Comment:
could you please add: faster than what?

Reply:
This is copied and I don't know faster that what :( Simply removed it. Perhaps @adbar could help